scripts/utils/getURLFromParams.js

```diff
@@ -1,13 +1,42 @@
+export const API_ENDPOINTS = {
+  GET_ALL_NODES: { version: 2, endpoint: '/get_all_nodes' },
+  GET_COLUMNS: { version: 2, endpoint: '/get_columns' },
+  GET_NODES: { version: 1, endpoint: '/get_nodes' },
+  GET_FLOWS: { version: 1, endpoint: '/get_flows' }
+};
+
+function getURLForV2(endpoint, params) {
+  return Object.keys(params).reduce((prev, current) => {
+    const value = params[current];
+    if (Array.isArray(value)) {
+      const arrUrl = value.reduce((arrPrev, arrCurrent) => {
+        return `${arrPrev}&${current}=${arrCurrent}`;
+      }, '');
+      return `${prev}&${arrUrl}`;
+    }
+    return `${prev}&${current}=${params[current]}`;
+  }, `${API_V2_URL}${endpoint}?`);
+}
+
 // builds an URL usable to call the API, using params
-export default (endpoint, params) => {
+function getURLForV1(endpoint, params) {
   return Object.keys(params).reduce((prev, current) => {
     const value = params[current];
     if (Array.isArray(value)) {
       const arrUrl = value.reduce((arrPrev, arrCurrent) => {
         return `${arrPrev}&${current}=${arrCurrent}`;
-      },'');
+      }, '');
       return `${prev}&${arrUrl}`;
     }
     return `${prev}&${current}=${params[current]}`;
-  }, `${API_URL}${endpoint}?`);
-};
+  }, `${API_V1_URL}${endpoint}?`);
+}
+
+export function getURLFromParams(endpointData, params) {
```

Comment:
Why not passing strings instead of object endpointData? API_ENDPOINTS should be internal and only the key should be needed